content/en/infrastructure/resource_catalog/policies/_index.md

```diff
@@ -30,30 +35,43 @@ Specifically, you can:
 
 ## Create a custom policy
 
-{{< img src="/infrastructure/resource_catalog/governance/custom-policy-example-1.png" alt="A custom policy reflecting a compliance score of thirty percent." width="100%">}}
+{{< img src="/infrastructure/resource_catalog/policies/custom-policy-example-2.png" alt="A custom policy reflecting a compliance score." width="100%">}}
```

Comment:
Just to confirm, is it intentional to have the `RESOURCE` values displayed in this image? It looks like these values are obfuscated in the [current image](https://docs.datadoghq.com/infrastructure/resource_catalog/governance/#create-a-custom-policy).

Also, just as a suggestion, it might be helpful to "zoom in"  the screenshot a bit more. Currently, it has to be expanded for the text to be legible.

Reply:
Good catch! Let me update this, those should not be visible, I'll get a new image, blur the values and enlarge it! Also good idea on using Pawparazzi in the future to remove "Preview" labels.